Identify releases PR in workflow

Address this error

```
Run PR_NUMBERS=$(gh pr list --state open --head releases/* --json number -q '.[].number')
no pull requests found for branch "develop"
Error: Process completed with exit code 1.
```

--- a/.github/workflows/finalize-release.yml
+++ b/.github/workflows/finalize-release.yml
@@ -20,7 +20,7 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          PR_NUMBERS=$(gh pr list --state open --head releases/* --json number -q '.[].number')
+          PR_NUMBERS=$(gh pr list --state open --base main --head releases/* --json number -q '.[].number')
           PR_COUNT=$(echo "$PR_NUMBERS" | wc -l)
           if [ "$PR_COUNT" -gt 1 ]; then
             echo "More than 1 release PR is open. Exiting workflow."

